package.json

```diff
@@ -8,8 +8,12 @@
     "url": "https://github.com/okgrow/merge-graphql-schemas.git"
   },
   "license": "MIT",
+  "engines": {
+    "node": ">=4.8.3",
+    "npm": ">=4.0.0"
+  },
   "scripts": {
-    "prepublish": ". ./scripts/prepublish.sh",
+    "prepare": ". ./scripts/prepublish.sh",
```

Comment:
@xavcz yes, `npm publish` & `npm install` will trigger the `pre-publish` in `npm 4` land. In `npm 5` pre-publish only runs on `publish`. See these two threads for more details https://github.com/npm/npm/issues/16685 https://github.com/npm/npm/issues/10074#issue-112707857